airflow_dag_samples/remote_dataset_dag/example_dag.py

```diff
@@ -78,7 +78,7 @@ def delete_volume(claim_name: str) -> None:
                                        "--secret_access_key", secret_access_key,
                                        "--bucket", "com-graphgrid-datasets",
                                        "--dataset_key",
-                                       "4l1heG30Et7NFUMO6ZCvphPhbzsvQKfvuBTWh4GGjntL/sample_dataset.jsonl",
```

Comment:
Out of curiosity why did we get rid of the string prefix here? Do you know what it was originally doing/why it was there?

Reply:
save_dataset creates a node in the graph representing a specified dataset, and also saves it out to minio with the dataset id (based on its node's grn) as the folder name. The dataset being passed in to get saved can be from anywhere (hence the different dag examples with different dataset locations), this one just happens to be in minio as well. The original dataset_key was an old dataset that was saved out with save_dataset a while ago, and no longer exists for me locally (I was just using it to test this dag before). I took out the dataset_id part of the dataset_key to hopefully clarify to users that the dataset can just simply be dropped in the specified bucket, rather than needing to be from a previously saved dataset.